sleap_nn/training/model_trainer.py

```
@@ -2,14 +2,17 @@
 
 from pathlib import Path
 from typing import Optional, List
+import functools
 import time
 from torch import nn
 import os
+import shutil
```

Ruff

`shutil` imported but unused

Remove unused import: `shutil`

(F401)